client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.test.js

```diff
@@ -48,6 +48,83 @@ describe('DataFilesToolbar', () => {
   });
 });
 
+describe('DataFilesToolbar', () => {
+  it('do not render unnecessary buttons in Community Data', () => {
+    const {getByText, queryByText} = renderComponent(
+      <DataFilesToolbar scheme="community" api="tapis" />,
+      mockStore({
+        workbench: { config: {
+          extract: '',
+          compress: ''
+        } },
+        files: { selected: { FilesListing: [] } },
+        listing: { selected: { FilesListing: [] } },
+        systems: systemsFixture
+      }),
+      createMemoryHistory()
+    );
+
+    expect(queryByText(/Rename/)).toBeFalsy();
+    expect(queryByText(/Move/)).toBeFalsy();
+    expect(getByText(/Copy/)).toBeDefined();
+    expect(getByText(/Download/)).toBeDefined();
+    expect(queryByText(/Trash/)).toBeFalsy();
+    expect(queryByText(/Make Public/)).toBeFalsy();
+  });
+});
+
+
+describe('DataFilesToolbar', () => {
```

Comment:
Could you combine all the tests that fall under `DataFilesToolbar` into one `describe` method?